public_html/wp-content/mu-plugins/blocks/assets/src/organizers/organizers-select.js

```diff
@@ -28,45 +28,47 @@ class OrganizersSelect extends Component {
 		};
 
 		this.buildSelectOptions = this.buildSelectOptions.bind( this );
-		this.fetchSelectOptions( props );
 	}
 
-	fetchSelectOptions( props ) {
+	static getDerivedStateFromProps( props, state ) {
 		const { allOrganizerPosts, allOrganizerTerms } = props;
 
-		const parsedPosts = allOrganizerPosts.then(
-			( fetchedPosts ) => {
-				const posts = fetchedPosts.map( ( post ) => {
-					return {
-						label  : post.title.rendered.trim() || __( '(Untitled)', 'wordcamporg' ),
-						value  : post.id,
-						type   : 'wcb_organizer',
-						avatar : post.avatar_urls[ '24' ],
-					};
-				} );
-
-				this.setState( { wcb_organizer: posts } );
-			}
-		);
+		if ( false === state.loading ) {
+			return;
```

Comment:
`SHOULD` - Does this need to return `null` instead of the implicit `undefined`? The docs say to use `null` to prevent state from being updated. `undefined` may work in practice too, but it feels safer to stick to the docs.

Reply:
This is no longer relevant after this refactor: 3df88a895deaefbb11a0958a5d340efd67f1a2e8

The component no longer uses state.